lib/r10k/git/shellgit.rb

```diff
@@ -53,6 +53,28 @@ def clean(opts = {})
         end
       end
 
+      def fetch(remote, opts = {})
+        cmd = ["fetch", remote, '+refs/*:refs/*']
+        result = git(cmd, opts)
+
+        if result.success?
+          return result.stdout.strip
```

Comment:
shellgit fetch and clone should return bools to be consistent with the rjgit provider...